app_dart/bin/gae_server.dart

```diff
@@ -18,23 +22,40 @@ Future<void> main() async {
     final Config config = Config(dbService, cache);
     final AuthenticationProvider authProvider = AuthenticationProvider(config: config);
     final AuthenticationProvider swarmingAuthProvider = SwarmingAuthenticationProvider(config: config);
+
     final BuildBucketClient buildBucketClient = BuildBucketClient(
       accessTokenService: AccessTokenService.defaultProvider(config),
     );
 
+    final BuildBucketV2Client buildBucketV2Client = BuildBucketV2Client(
+      accessTokenService: AccessTokenService.defaultProvider(config),
+    );
+
     /// LUCI service class to communicate with buildBucket service.
     final LuciBuildService luciBuildService = LuciBuildService(
       config: config,
       cache: cache,
       buildBucketClient: buildBucketClient,
+      buildBucketV2Client: buildBucketV2Client,
       pubsub: const PubSub(),
     );
 
+    // final LuciBuildServiceV2 luciBuildServiceV2 = LuciBuildServiceV2(
+    //   config: config,
+    //   cache: cache,
+    //   buildBucketV2Client: buildBucketV2Client,
+    //   pubsub: const PubSub(),
+    // );
```

Comment:
Are these logics intended to make the following up PR easy? I am okay with leaving them here if we get to following PRs in coming days.

Reply:
I kept them included to remind myself. It was easier this way as I needed to modify the source in order to get a clean run.